packages.yml

```diff
@@ -4,3 +4,6 @@ packages:
 
 - package: dbt-labs/spark_utils
   version: [">=0.3.0", "<0.4.0"]
+
+- package: calogica/dbt_date
+  version: [">=0.10.0", "<0.11.0"]
```

Comment:
Let's expand this to be `<1.0.0`. Similar to what we did in the [latest Zendesk update](https://github.com/fivetran/dbt_zendesk/blob/62df5b5fee7e448357f92c007c8212a0d6ece2f7/packages.yml#L6). We confirmed with Calogica that this would be appropriate and helps to reduce the dependency conflicts some may face in the future.